sci-libs/tensorflow/files/tensorflow-2.0.0-jsoncpp-1.9.2.patch

```diff
@@ -0,0 +1,26 @@
+diff --git a/third_party/systemlibs/jsoncpp.BUILD b/third_party/systemlibs/jsoncpp.BUILD
+index 7d54f92..ce14265 100644
+--- a/third_party/systemlibs/jsoncpp.BUILD
++++ b/third_party/systemlibs/jsoncpp.BUILD
+@@ -10,7 +10,7 @@ HEADERS = [
+     "include/json/assertions.h",
+     "include/json/autolink.h",
+     "include/json/config.h",
+-    "include/json/features.h",
```

Comment:
Do they really need all those headers? `json.h` includes all of them, so this really looks like another clueless upstream.

Reply:
The way they do things, they do. The include/json directory is not added to the includepath, the relevant include files are softlinked to the build tree instead. Softlinking only the json/json.h would not tell the build system where to find the rest of the includes.

This is probably to make the process essentially the same as it is for the bundled jsoncpp, but it is demostrably fragile as it broke with 1.9 and again with 1.9.2.

Reply:
/me facepalms to the floor level.